packages/office-ui-fabric-react/src/components/pickers/BasePicker.tsx

```diff
@@ -180,6 +180,15 @@ export class BasePicker<T, P extends IBasePickerProps<T>> extends BaseComponent<
     );
   }
 
+  protected okToAddMore(): boolean {
+    const { items } = this.state;
+    const { itemLimit } = this.props;
+    if (typeof itemLimit !== 'undefined' && items.length === itemLimit) {
```

Comment:
I tried the shortest version. 
`javascript
return items.length < itemLimit;
`
and it seems to break existing functionality, when running the tests, as itemLimit is not passed in in most cases.
I think we should go with 
```javascript
return itemLimit === undefined || items.length < itemLimit;
```
so, when its undefined, its `canAddMore() === true` 
and when its defined, `items.length` has to be less than itemLimit to return true..